hooks/useUserStateSyncedWithCookies.tsx

```diff
@@ -5,10 +5,10 @@ import {
   COOKIE_NAME_ADULT_FILTER,
   COOKIE_NAME_NEW_TAB,
   COOKIE_NAME_SEARCH_COUNT,
+  getCookie,
+  CookieMap,
+  setCookie,
 } from '../helpers/_cookies'
-import Cookies from 'js-cookie'
-
-type CookieMap = { [cookieName: string]: string }
 
 const cookiesName = {
```

Comment:
we could move this to the cookies helper file too, what do you think?

Reply:
I'd say this is more related to the user state than to the cookies. Here we merge both things. A better name for this would be something like `userStateToCookieNameMap` which shows why it's not an independent cookie topic.